crates/pathfinder/src/sierra.rs

```diff
@@ -232,21 +230,28 @@ mod tests {
 
     mod starknet_v0_11_2_onwards {
         use super::*;
-        use starknet_gateway_test_fixtures::class_definitions::CAIRO_1_1_0_RC0_SIERRA;
+        use starknet_gateway_test_fixtures::class_definitions::{
+            CAIRO_1_1_0_RC0_SIERRA, CAIRO_2_0_0_STACK_OVERFLOW,
+        };
 
         #[test]
         fn test_feeder_gateway_contract_conversion() {
             let class =
                 serde_json::from_slice::<FeederGatewayContractClass<'_>>(CAIRO_1_1_0_RC0_SIERRA)
                     .unwrap();
 
-            let _: casm_compiler_v1_0_0_rc0::contract_class::ContractClass =
-                class.try_into().unwrap();
+            let _: casm_compiler_v2::contract_class::ContractClass = class.try_into().unwrap();
         }
 
         #[test]
         fn test_compile() {
             compile_to_casm(CAIRO_1_1_0_RC0_SIERRA, &StarknetVersion::new(0, 11, 2)).unwrap();
         }
+
+        #[tokio::test]
+        async fn regression_stack_overflow() {
```

Comment:
why `async`?

Reply:
Ah thanks, originally I had the definition pulled from the gateway :D